modules/channel-messenger/src/backend/messenger.ts

```diff
@@ -159,8 +159,13 @@ export class MessengerService {
         debugMessages('could not find a bot for page id =', pageId)
         continue
       }
-
+      if (!messages || !messages.sender) {
```

Comment:
this validation was incorrect, messages is an array so `messages.sender` ain't valid